maint-lib/stage.py

```diff
@@ -45,10 +45,26 @@
     sys.exit(1)
 
 
+def populateCephPointRelease():
+    # We need to extract a possible CEPH_POINT_RELEASE from CEPH_VERSION
+    # It could be under the two following syntax :
+    # - package_name-package_version
+    # - package_name=package_version"
+    os.environ["CEPH_POINT_RELEASE"] = ""
+    if os.environ["CEPH_VERSION"]:
+        for separator in ["-", "="]:
+            if separator in os.environ["CEPH_VERSION"]:
+                version, point_release = os.environ["CEPH_VERSION"].split(separator)
```

Comment:
This may split into more than 2 pieces if the point release spec has dashes (e.g., `ceph-12.2.2-5` which would indicate ceph v12.2.2 build 5). This should handle that and assume that the version name is before the first dash and that everything after it belongs to `point_release`.

```py
>>> vers
'luminous-12.2.2-5'
>>> vers.split('-')
['luminous', '12.2.2', '5']
>>> v, p = vers.split('-')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: too many values to unpack (expected 2)
```

Reply:
```py
>>> splitvers = vers.split('-')
>>> v = splitvers[0]
>>> v
>>> p = '-'.join(splitvers[1:])
>>> p
'12.2.2-5'
```

Reply:
Fixed.